docker-compose.yml

```diff
@@ -36,6 +36,7 @@ services:
      AWS_SECRET_ACCESS_KEY: "${AWS_SECRET_ACCESS_KEY}"
      ENV: "${ENV:-development}"
      FLASK_APP: OpenOversight.app
+     OO_ADMIN_EMAIL: "info@lucyparsonslabs.com"
```

Comment:
I'm not sure if we need to use a different email address, but this is the one that made the most sense to use at the time I created the functionality.